driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -80,23 +81,53 @@ public ChangeStreamDocument(@BsonProperty("resumeToken") final BsonDocument resu
      * @param operationType the operation type
      * @param updateDescription the update description
      */
-    @BsonCreator
+    @Deprecated
     public ChangeStreamDocument(@BsonProperty("resumeToken") final BsonDocument resumeToken,
```

Comment:
Since it's no longer annotated with BsonCreator, remove all the parameter annotations.

Reply:
Might as well do that with the other deprecated constructor too

Reply:
Done